pkg/cmd/release/create/create.go

```diff
@@ -224,6 +228,18 @@ func createRun(opts *CreateOptions) error {
 		}
 	}
 
+	var remoteTagPresent bool
```

Comment:
This variable can now be pushed into the following `if` block as its scope because it won't be needed elsewhere